api/backupbackingimage.go

```diff
@@ -71,8 +72,9 @@ func (s *Server) BackupBackingImageCreate(w http.ResponseWriter, req *http.Reque
 	}
 
 	backingImageName := mux.Vars(req)["name"]
-	if err := s.m.CreateBackupBackingImage(input.Name, backingImageName, input.BackupTargetName); err != nil {
-		return errors.Wrapf(err, "failed to create backup backing image '%s'", input.Name)
+	backupBackingImageName := types.GetBackupBackingImageNameFromBIName(backingImageName)
```

Comment:
- Can we check if `input.Name` is equal to `backingImageName` or an empty string, and then generate a random UUID name?
- Or we won't allow users to customize the backup backing image name from the API when creating it.

Reply:
cc @mantissahz @derekbit 
We don't allow customizing the backup backing image name

The name is directly from the URL
`HTTP/1.1 POST /v1/backingimages/parrot?action=backupBackingImageCreate`
```
	r.Methods("DELETE").Path("/v1/backingimages/{name}").Handler(f(schemas, s.BackingImageDelete))
	backingImageActions := map[string]func(http.ResponseWriter, *http.Request) error{
		"backupBackingImageCreate": s.BackupBackingImageCreate,
	}
```

Just like we don't allow customizing the backup volume name

Reply:
Got it. It's reasonable.